src/Illuminate/Contracts/Pipeline/Pipeline.php

```diff
@@ -7,23 +7,23 @@
 interface Pipeline
 {
     /**
-     * Set the traveler object being sent on the pipeline.
+     * Set the object being sent through the pipeline.
```

Comment:
I copied mostly from the Pipeline and Route Pipeline impl. 
They were all committed on the same commit/day I think